index.html

```diff
@@ -0,0 +1,416 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>KMarketer</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body>
+    <!-- Header - Mobile -->
+    <header class="header header--mobile-screen">
+        <div class="header__container">
+            <h1 class="logo">KMarketer</h1>
+            <div class="header__icons">
+                <img src="./assets/search-icon.png" alt="Search Icon" class="header__icon">
+                <img src="./assets/cart-icon.png" alt="Cart Icon" class="header__icon">
+                <img src="./assets/hamburger-icon.png" alt="Hamburger Icon" class="header__icon"> 
+            </div>
+        </div>
+        <ul class="links">
+            <li class="links__item">Home</li>
+            <li class="links__item">Product</li>
+            <li class="links__item">Pricing</li>
+            <li class="links__item">Contact</li>
+        </ul>
+    </header>
+    <!-- Header - Mobile -->
+
+    <!-- Header - Desktop -->
+    <header class="header header--desktop-screen">
+        <div class="header__container">
+            <h1 class="logo">KMarketer</h1>
+            <div class="header__items">
+                <div>
+                    <ul class="links links--row">
+                        <li class="links__item">Home</li>
+                        <li class="links__item">Product</li>
+                        <li class="links__item">Pricing</li>
+                        <li class="links__item">Contact</li>
+                        <li>
+                            <img src="./assets/search-icon.png" alt="Search Icon" class="header__search-icon">
+                        </li>
+                        <li>
+                            <img src="./assets/cart-icon.png" alt="Cart Icon" class="header__cart-icon header__cart-con--desktop">
```

Comment:
You have a typo here on your element `cart-con`, I think you meant `cart-icon` because that's the element you have there.